charts/harvester-cloud-provider/values.yaml

```diff
@@ -11,15 +11,13 @@ image:
   tag: v0.1.1
 
 clusterName: ""
-cloudConfigPath: "/var/lib/rancher/rke2/etc/config-files/cloud-provider-config"
+cloudConfigPath: "/etc/kubernetes/cloud-config"
```

Comment:
I suggest we use the rke2 path as the default

Reply:
Set the rke1 path as the default and add the questions.yml